tracing-opentelemetry/src/subscriber.rs

```diff
@@ -536,16 +540,18 @@ where
         let mut extensions = span.extensions_mut();
 
         if let Some(mut builder) = extensions.remove::<otel::SpanBuilder>() {
-            // Append busy/idle timings when enabled.
-            if let Some(timings) = extensions.get_mut::<Timings>() {
-                let mut timings_attributes = vec![
-                    KeyValue::new("busy_ns", timings.busy.to_string()),
-                    KeyValue::new("idle_ns", timings.idle.to_string()),
-                ];
-
-                match builder.attributes {
-                    Some(ref mut attributes) => attributes.append(&mut timings_attributes),
-                    None => builder.attributes = Some(timings_attributes),
+            if self.tracked_inactivity {
+                // Append busy/idle timings when enabled.
+                if let Some(timings) = extensions.get_mut::<Timings>() {
+                    let mut timings_attributes = vec![
+                        KeyValue::new("busy_ns", timings.busy.to_string()),
+                        KeyValue::new("idle_ns", timings.idle.to_string()),
+                    ];
+
+                    match builder.attributes {
+                        Some(ref mut attributes) => attributes.append(&mut timings_attributes),
+                        None => builder.attributes = Some(timings_attributes),
```

Comment:
nit/TIOLI: it might be possible to avoid an allocation here when the attributes vec is already non-empty? but we can save that for a future PR...

Reply:
Good catch, also recording these values as i64 instead of converting to string when exporting, shave off a few more allocations (spec disallows recording u64 values directly currently 🤷‍♂️)